MotionMark/tests/resources/math.js

```diff
@@ -22,14 +22,17 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
-SimpleKalmanEstimator = Utilities.createSubclass(Experiment,
-    function(processError, measurementError) {
-        Experiment.call(this, false);
+
+class SimpleKalmanEstimator extends Experiment {
```

Comment:
I think we need to get rid of this `Kalman` filter as well. The `Regression` class tries to find the best fit two piece regression. So we should not worry about lessening the noise of the samples.

Reply:
Let's do functional changes separately.